print/templates/pattern_only.html

```diff
@@ -0,0 +1,25 @@
+    <main class="page pattern-onlyview" id="pattern-display">
+      <figure class="inksimulation pattern-only operator"
+              data-field-name="pattern-onlyview"
+              title="{{ _('Ctrl + Scroll to Zoom') }}"
+              style="cursor: auto;">
+                {{ svg_overview|replace("<li>", "")|replace("</li>", "")|safe }}
+                {% with %}
+                {% set realistic_id='realistic-operator-overview' %}
```

Comment:
We need a specific id here which should not be repeated. Also set the sizing buttons with a template right there.

```
                {% set realistic_id='realistic-pattern-onlyview' %}
                {% include 'ui_svg_action_buttons.html' with context %}
```